src/xdocs/checks.xml

```diff
@@ -898,14 +910,17 @@
       to gather information the filter can't get on it's own.
     </p>
 
-    <table>
-      <tr>
-        <td><a href="config_annotation.html#SuppressWarningsHolder">SuppressWarningsHolder</a></td>
-        <td>
-          Maintains a set of check suppressions from <code>SuppressWarnings</code> annotations.
-        </td>
-      </tr>
-    </table>
+      <div class="wrapper">
```

Comment:
same.

Reply:
the div has one more indent here because the table had the same indent as the wrapping section.